src/components/organisms/searchResult/SearchResult.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import ItemList from '../../molecules/itemList/ItemList'
+import UserInfo from '../../molecules/userInfo/UserInfo'
+
+export default function SearchResult(props) {
+
+  const starCount = (ItemList) => {
+    let sum = 0
+    ItemList.map(({starCount}) => {
+      sum += starCount
+    })
+    return sum

```

Comment:
위와같은 케이스는 map을 사용하기보단 reduce를 사용하신다면 의도를 더 드러내기 좋을 것 같습니다!
```suggestion
   return ItemList.reduce((acc, {starCount}) =>  acc += starCount, 0);
```

Reply:
이렇게 간단하게 될줄은 몰랐어요 ㅠㅠ 감사합니다!